src/ml/neural_net/style_transfer/mps_style_transfer_encoding_node.h

```diff
@@ -0,0 +1,24 @@
+#import <Accelerate/Accelerate.h>
+#import <Foundation/Foundation.h>
+#import <Metal/Metal.h>
+#import <MetalPerformanceShaders/MetalPerformanceShaders.h> 
+
+#include <ml/neural_net/style_transfer/mps_style_transfer_utils.h>
```

Comment:
Use #import for ObjC headers.

FWIW, I've been trying to move towards .h for ObjC (or pure C), and .hpp for C++ headers